build/configs/artik053/tc/defconfig

```diff
@@ -240,9 +240,9 @@ CONFIG_ARTIK05X_FLASH_CAPACITY=8388608
 CONFIG_ARTIK05X_FLASH_PAGE_SIZE=4096
 CONFIG_ARTIK05X_FLASH_PART=y
 CONFIG_ARTIK05X_FLASH_MINOR=0
-CONFIG_ARTIK05X_FLASH_PART_LIST="16,48,192,32,512,2400,1536,1536,1400,8,512,"
-CONFIG_ARTIK05X_FLASH_PART_TYPE="none,ftl,none,none,none,none,none,ftl,smartfs,config,none,"
-CONFIG_ARTIK05X_FLASH_PART_NAME="bl1,sssro,bl2,sssfw,wlanfw,os,factory,ota,user,nvram,sssrw,"
+CONFIG_ARTIK05X_FLASH_PART_LIST="16,48,192,32,512,2400,1536,1536,1000,400,8,512,"
```

Comment:
I can't find anywhere you use romfs for this example. Could you let me know why rom is needed?